src/EnergyPlus/DaylightingManager.cc

```diff
@@ -4381,59 +4394,60 @@ void GetDaylightingControls(EnergyPlusData &state, bool &ErrorsFound)
         auto &daylightControl = dl->daylightControl(controlNum);
         daylightControl.Name = ipsc->cAlphaArgs(1);
 
-        // Is it a space or zone name?
-        int const spaceNum = Util::FindItemInList(state.dataIPShortCut->cAlphaArgs(2), state.dataHeatBal->space);
-        if (spaceNum > 0) {
-            daylightControl.spaceIndex = spaceNum;
-            daylightControl.zoneIndex = state.dataHeatBal->space(spaceNum).zoneNum;
-            daylightControl.enclIndex = state.dataHeatBal->space(spaceNum).solarEnclosureNum;
-            // Check if this is a duplicate
-            if (spaceHasDaylightingControl(spaceNum)) {
-                ShowSevereError(state,
-                                format("{}=\"{}\" Space==\"{}\" already has a {} object assigned to it. Only one per Space is allowed.",
-                                       ipsc->cCurrentModuleObject,
-                                       daylightControl.Name,
-                                       state.dataHeatBal->space(spaceNum).Name,
-                                       ipsc->cCurrentModuleObject));
-                ErrorsFound = true;
-                continue;
+        // Is it a zone or space name?
```

Comment:
Reverse the order for consistency with other objects, check for zone name match first, then space name.